sdks/python/tox.ini

```diff
@@ -290,7 +290,7 @@ commands =
 [testenv:docs]
```

Comment:
Please rename to py37-docs. This will tell tox to use Python 3.7 instead of the current default.

Reply:
tox.ini and build.gradle invocating `docs` command are both updated.